tests/kafkatest/tests/core/compatibility_test_new_broker_test.py

```diff
@@ -46,12 +45,12 @@ def setUp(self):
 
     @cluster(num_nodes=6)
     @parametrize(producer_version=str(LATEST_0_8_2), consumer_version=str(LATEST_0_8_2), compression_types=["none"], new_consumer=False, timestamp_type=None)
-    @parametrize(producer_version=str(LATEST_0_8_2), consumer_version=str(LATEST_0_9), compression_types=["none"], new_consumer=False, timestamp_type=None)
```

Comment:
Yeah, I guess I just don't know what the strategy is for this test in terms of how we are choosing the combinations of producer/consumer version. Which makes it hard to review a patch adding/removing parameterizations :)